src/main/java/seedu/intrack/logic/commands/TaskCommand.java

```diff
@@ -0,0 +1,90 @@
+package seedu.intrack.logic.commands;
+
+import static seedu.intrack.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.intrack.model.Model.PREDICATE_SHOW_ALL_INTERNSHIPS;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import seedu.intrack.commons.core.Messages;
+import seedu.intrack.commons.core.index.Index;
+import seedu.intrack.logic.commands.exceptions.CommandException;
+import seedu.intrack.model.Model;
+import seedu.intrack.model.internship.Internship;
+import seedu.intrack.model.internship.Task;
+
+/**
+ * Updates the current task of an Internship.
+ */
+public class TaskCommand extends Command {
+
+    public static final String COMMAND_WORD = "task";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Updates the task of the internship identified by "
+            + "the index number used in the displayed list.\n"
+            + "Parameters: INDEX (must be a positive integer) "
+            + " TASK\n"
+            + "Example: " + COMMAND_WORD + " 1 Technical Interview /at 04-11-2022(17:00)";
+
+    public static final String TASK_COMMAND_CONSTRAINTS = "TASK must be in the format: \n"
+            + COMMAND_WORD + " INDEX DESCRIPTION /at TIME\n"
+            + "TIME must be in the format dd-MM-yyyy(HH:mm)";
```

Comment:
I think the brackets are unnecessary and could be a hassle for users to type out?
Perhaps a simpler format like `dd-MM-yyyy HH:mm` could be used?

Reply:
Thank you for the advice, I have changed the format to be more convenient for users.